core-bundle/src/Resources/contao/library/Contao/StringUtil.php

```diff
@@ -635,6 +635,67 @@ public static function insertTagToSrc($data)
 		return $return;
 	}
 
+	/**
+	 * Adds {{env::base_path}} to relative links
+	 *
+	 * @param string $data The markup string
+	 *
+	 * @return string
+	 */
+	public static function addBasePath($data)
```

Comment:
we should probably have unit tests for these new methods?

Reply:
We already do.